src/ripple/nodestore/impl/DeterministicShard.h

```diff
@@ -61,7 +61,8 @@ class DeterministicShard
         DatabaseShard const& db,
         std::uint32_t index,
         uint256 const& lastHash,
-        beast::Journal j);
+        beast::Journal j,
+        boost::filesystem::path const& dir);
```

Comment:
`journal` should be the last parameter

Reply:
I think this will get removed since `DeterministicShards` are getting pulled from the release.

Reply:
Removed.